persist-cli/src/main/java/io/ballerina/persist/cmd/Push.java

```diff
@@ -182,36 +190,35 @@ public void execute() {
                     ScriptRunner sr = new ScriptRunner(connection);
                     sr.runQuery(query);
                 } catch (SQLException e) {
-                    errStream.println("ERROR: failed to create the database(" +
-                            persistConfigurations.getDbConfig().getDatabase() + "). " + e.getMessage());
+                    errStream.printf("ERROR: failed to create the database(%s). %s%n",
+                            persistConfigurations.getDbConfig().getDatabase(), e.getMessage());
                     return;
                 }
-                errStream.println("Created database `" + persistConfigurations.getDbConfig().getDatabase() + "`.");
+                errStream.printf("Created database '%s'.", persistConfigurations.getDbConfig().getDatabase());
```

Comment:
new line character is missing

Reply:
updated in [4a4bd31](https://github.com/ballerina-platform/persist-tools/pull/140/commits/4a4bd31fb65a4ef430fc372c49473ee14bd4eaac)